src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -907,7 +870,6 @@ await API.graphql({
 ##### Compound filters
 
 You can combine filters with `and`, `or`, and `not` Boolean logic. Observe, in the autogenerated schema above, that `ModelTodoFilterInput` is recursive in respect to those fields. So if, for example, you wanted to filter for `priority` values of 1 OR 2, you would do this:
-[comment]: # (HEATHER, this paragraph references an earlier section. We'll need to discuss how we want to handle this in terms of the modularization approach.)
```

Comment:
Just noting there was a Q about reference to earlier section and impact on modularization - this all looks to be part of the same H2 so we should be good